Libraries/Components/Button.windows.js

```diff
@@ -84,7 +91,27 @@ class Button extends React.Component {
      * Used to locate this view in end-to-end tests.
      */
     testID: PropTypes.string,
-  };
+    /**
+     * (Windows only) tabIndex (default is 0 rather than undefined)
```

Comment:
>tabIndex (default is 0 rather than undefined) [](start = 22, length = 45)

What values are permitted for tab indices and what are the semantics? If it maps directly to the UWP semantics for tab index, it'll be helpful to describe here anyway for those that aren't familiar with UWP. Also, a description of the contract will be helpful in case anybody wants to extend this to another platform.

Reply:
I'll comment. It's just the FocusableView that follows the UWP convention (tabIndex + IsTabStop combo) since it's something very custom to the UWP implementation of React Native.

For Button & TextInput it's something that mimics in a way ReactXP.
-1 not focusable at all (not even by clicking, programatically, etc.). Making it work like in DOM wouldn't be trivial.
0 and up: same as DOM